mutalyzer/describe.py

```diff
@@ -185,653 +53,344 @@ def makeFSTables(table_id):
         reverse_table[table[i]].append(i)
 
     # Make the frame shift tables.
-    FS1 = collections.defaultdict(set)
-    FS2 = collections.defaultdict(set)
-    for AA_i in reverse_table:
-        for AA_j in reverse_table:
-            for codon_i in reverse_table[AA_i]:
-                for codon_j in reverse_table[AA_j]:
-                    FS1[AA_i + AA_j].add(table[(codon_i + codon_j)[1:4]]) # +1.
-                    FS2[AA_i + AA_j].add(table[(codon_i + codon_j)[2:5]]) # +2.
+    fs1 = collections.defaultdict(set)
+    fs2 = collections.defaultdict(set)
+    for aa_i in reverse_table:
+        for aa_j in reverse_table:
+            for codon_i in reverse_table[aa_i]:
+                for codon_j in reverse_table[aa_j]:
+                    fs1[aa_i + aa_j].add(table[(codon_i + codon_j)[1:4]]) # +1.
+                    fs2[aa_i + aa_j].add(table[(codon_i + codon_j)[2:5]]) # +2.
                 #for
-    return FS1, FS2
-#makeFSTables
+    return fs1, fs2
+#make_fs_tables
 
-def __makeOverlaps(peptide):
+def _peptide_overlaps(peptide):
     """
     Make a list of overlapping 2-mers of {peptide} in order of appearance.
 
-    @arg peptide: A peptide sequence.
-    @type peptide: unicode
-    @returns: All 2-mers of {peptide} in order of appearance.
-    @rtype: list(unicode)
+    :arg peptide: A peptide sequence.
+    :type peptide: unicode
+
+    :returns: All 2-mers of {peptide} in order of appearance.
+    :rtype: list(unicode)
     """
     return map(lambda x: peptide[x:x+2], range(len(peptide) - 1))
```

Comment:
Prefered style is

    return [peptide[x:x+2] for x in range(len(peptide) - 1)]

or

    return [a + b for (a, b) in zip(peptide, peptide[1:])]

which are also faster.